src/scp/SCP.cpp

```diff
@@ -120,28 +120,93 @@ SCP::getJsonInfo(size_t limit, bool fullKeys)
     return ret;
 }
 
+SCP::QuorumInfoNodeState
+SCP::getState(NodeID const& node, uint64 slotIndex, int N = 2)
```

Comment:
don't use this N=2 (I would remove this argument instead): this function is only called from one place and that syntax is bogus anyways and should be removed : I am not even sure why this compiles (default arguments should be defined in the signature ie the ".h" file).

Reply:
Removed. I turned this into a constant.